src/pages/__test__/product.test.js

```diff
@@ -0,0 +1,2861 @@
+                                        url: 'https://images.prismic.io/wizeline-academy/3f987996-cf7e-458f-82ba-c530eb10f3cd_2.jpeg?auto=compress,format'
+                                    }
+                                },
+                                {
+                                    image: {
+                                        dimensions: {
+                                            width: 940,
+                                            height: 1215
+                                        },
+                                        alt: null,
+                                        copyright: null,
+                                        url: 'https://images.prismic.io/wizeline-academy/f74990c9-5b99-43f0-ba16-2db801ecf58a_3.webp?auto=compress,format'
+                                    }
+                                },
+                                {
+                                    image: {
+                                        dimensions: {
+                                            width: 940,
+                                            height: 1215
+                                        },
+                                        alt: null,
+                                        copyright: null,
+                                        url: 'https://images.prismic.io/wizeline-academy/7ac48c95-3dad-4124-86b3-909fa14eaf54_4.jpeg?auto=compress,format'
+                                    }
+                                },
+                                {
+                                    image: {
+                                        dimensions: {
+                                            width: 940,
+                                            height: 1215
+                                        },
+                                        alt: null,
+                                        copyright: null,
+                                        url: 'https://images.prismic.io/wizeline-academy/72e1dd40-3880-4485-9bfd-5b9042330414_5.webp?auto=compress,format'
+                                    }
+                                }
+                            ],
+                            stock: 22,
+                            price: 106
+                        }
+                    }
+                ],
+                version: '73ae4f1',
+                license:
+                    'This work is licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/.'
+            }
+        });
+
+        render(
+            <AppContext.Provider value={mockDispatch}>
+                <Router>
+                    <ProductsPage />
+                </Router>
+            </AppContext.Provider>
+        );
+        fireEvent.click(screen.getByTestId('cat-Decorate'));
+        await waitFor(async () => {
+            const products = await screen.findByTestId('productsList');
```

Comment:
As you are using the `find` selector you don't need the `waitFor` so please remove it, only leave the find one.

Reply:
Same comment for the other tests.